ci/requirements.txt

```diff
@@ -1,7 +1,7 @@
 numpy>=1.17.2, <1.23.1
-torch>=1.10, <=1.13.0
-torchvision>=0.11, <=0.14.0
-pytorch-lightning>=1.5.9, <1.7.0  # refactor code, if you want newer versions
+torch>=2.0
+pytorch-lightning>=2.1.0
```

Comment:
pytorch-lightning follow torch version, so pytorch-lightning>=2.0

Reply:
okay, let's try to run tests for lightning >= 2.0

Reply:
in CI

Reply:
seems like it works